pretty_midi/instrument.py

```diff
@@ -113,6 +120,29 @@ def get_piano_roll(self, fs=100, times=None):
             piano_roll[note.pitch,
                        int(note.start*fs):int(note.end*fs)] += note.velocity
 
+        # Process sustain pedals
+        if pedal_threshold is not None:
+            CC_SUSTAIN_PEDAL = 64
+            time_pedal_on = 0
+            is_pedal_on = False
+            for cc in [_e for _e in self.control_changes
+                       if _e.number == CC_SUSTAIN_PEDAL]:
+                time_now = int(cc.time*fs)
+                is_current_pedal_on = (cc.value > pedal_threshold)
```

Comment:
Very minor nit, >= 64 seems like a better threshold to me because at 64, you're in the upper half of values.

Likely doesn't matter at all because for this kind of sustain processing, we more or less expect it to be just on or off.

Reply:
Thanks for the good point! I fixed the code accordingly.